apps/user-office-backend/src/eventHandlers/messageBroker.ts

```diff
@@ -45,17 +48,17 @@ type ProposalMessageData = {
 
 let rabbitMQCachedBroker: null | RabbitMQMessageBroker = null;
 
-const getRabbitMQMessageBroker = () => {
+const getRabbitMQMessageBroker = async () => {
   if (rabbitMQCachedBroker === null) {
-    rabbitMQCachedBroker = createRabbitMQMessageBroker();
+    rabbitMQCachedBroker = await createRabbitMQMessageBroker();
   }
 
   return rabbitMQCachedBroker;
 };
 
-const createRabbitMQMessageBroker = () => {
+const createRabbitMQMessageBroker = async () => {
   const messageBroker = new RabbitMQMessageBroker();
-  messageBroker.setup({
+  await messageBroker.setup({
     hostname: process.env.RABBITMQ_HOSTNAME,
     username: process.env.RABBITMQ_USERNAME,
     password: process.env.RABBITMQ_PASSWORD,
```

Comment:
Shouldn't we use try catch for this block of code? I assume it tries to connect to something, but what if it fails? Does it throw error or?